### PR TITLE
DB/Script : Zone - The Wandering Isle

### PR DESCRIPTION
**Pull request by shamage**
Co-authored-by: Aquadeus <efectokoleos@gmail.com>

**Changes proposed:**

-  
-  
-  

**Issues addressed:**

Closes #  (insert issue tracker number)

**Tests performed:**

(Does it build, tested in-game, etc.)

**Known issues and TODO list:** (add/remove lines as needed)

- [ ] 
- [ ]

### CODE REVIEW

**Comment by Aokromes** [comment]
/home/runner/work/TrinityCore/TrinityCore/src/server/scripts/Pandaria/zone_the_wandering_isle.cpp:59:64: error: unused parameter ‘task’ [-Werror=unused-parameter]
             _scheduler.Schedule(Seconds(1), [this](TaskContext task)

**Comment by Faq** [comment]
Why not go with sai?

**Comment by MaxtorCoder** [comment]
PR title is incorrect, code style is wrong. :/

**Comment by shamage** [comment]
> Prečo neísť so sai?



> Why not go with sai?

I tried it, but it didn't work for all the actions that were supposed to take place during the session - mainly killkredit could not be obtained without killing the NPC

**Comment by shamage** [comment]
> PR title is incorrect, code style is wrong. :/

what exactly PR name should I give it?

File: src/server/skripty/Pandaria/pandaria_script_loader.cpp [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/*
```
Comment (Aokromes) [inline src/server/skripty/Pandaria/pandaria_script_loader.cpp:1]: src/server/skripty/Pandaria/pandaria_script_loader.cpp
bad name

**Comment by meji46** [comment]
> > Why not go with sai?
> 
> I tried it, but it didn't work for all the actions that were supposed to take place during the session - mainly killkredit could not be obtained without killing the NPC

You can use SMART_ACTION_CALL_KILLEDMONSTER

**Comment by Aokromes** [comment]
/home/runner/work/TrinityCore/TrinityCore/src/server/scripts/Pandaria/zone_the_wandering_isle.cpp:59:64: error: unused parameter ‘task’ [-Werror=unused-parameter]
_scheduler.Schedule(Seconds(1), [this](TaskContext task)

still valid.

**Comment by Aokromes** [comment]
/home/runner/work/TrinityCore/TrinityCore/src/server/scripts/Pandaria/zone_the_wandering_isle.cpp:64:64: error: unused parameter ‘task’ [-Werror=unused-parameter]
             _scheduler.Schedule(Seconds(3), [this](TaskContext task)
                                                    ~~~~~~~~~~~~^~~~

File: src/server/scripts/Pandaria/zone_the_wandering_isle.cpp [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                    Talk(SAY_FINISH_FIGHT);
+                });
+
+            _scheduler.Schedule(Seconds(3), [this](TaskContext task)
+                {
+                    // ToDo: There are 3 or 4 points where the npc goes randomly
+                    me->GetMotionMaster()->MovePoint(0, 1446.302f, 3387.493f, 173.7903f);
+                });
+
+            _scheduler.Schedule(Seconds(6), [this](TaskContext task)
```
Comment (Aokromes) [inline src/server/scripts/Pandaria/zone_the_wandering_isle.cpp:70]: i supose this needs /* */ too

**Comment by mdX7** [comment]
NOTE: Just realized that the path the npc runs to depends on where the npc stands when he hits <=20% (aka becomes IMMUNE_TO_PC). So we could always use the same npc to figure out where the boundaries to the next path are

**Pull request closed without merging**